src/test/java/seedu/address/logic/LogicManagerTest.java

```diff
@@ -67,7 +67,7 @@
     private int targetedJumpIndex;
 
     @Subscribe
-    private void handleLocalModelChangedEvent(AddressBookChangedEvent abce) {
+    private void handleLocalModelChangedEvent(TaskManagerChangedEvent abce) {
```

Comment:
`abce` stands for `AddressBookChangedEvent`, perhaps `tmce` would make more sense here.